lib/ello_click/affiliate/link.ex

```diff
@@ -0,0 +1,9 @@
+defmodule ElloClick.Affiliate.Link do
+  defstruct [original: nil, affiliated: nil, is_affiliated: false]
```

Comment:
I don't mind the shorter names. Since they are in a `Link` struct the "link" prefix is not needed. Not sure what you think @alanpeabody.